src/content/pages/security.md

```diff
@@ -0,0 +1,8 @@
+---
+header:
+  title: "Filecoin Ecosystem Security"
+  description: "Filecoin Foundation is deeply committed to the integrity and security of the Filecoin network. Robust security practices are vital for developing, maintaining, and operating resilient systems – enabling the Filecoin community to innovate and grow."
+seo:
+  title: "Filecoin Ecosystem Security"
+  description: "Filecoin Foundation is deeply committed to the integrity and security of the Filecoin network. Robust security practices are vital for developing, maintaining, and operating resilient systems – enabling the Filecoin community to innovate and grow."
```

Comment:
Once the page is up, we will ask David to provide an appropriate SEO title and description. Can be in the same batch as Orbit page.

Reply:
So we leave as is for now?